documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1537,6 +1556,11 @@ For details about setting up Oracle for use with the {prodname} connector, see t
 
 endif::product[]
 
+=== Compatibility with Oracle installation types
+
+An Oracle database can be installed either as a standalone instance or using Oracle Real Application Cluster (RAC).
```

Comment:
We shouldn't describe RAC as supported until that's verified by QE. I.e. this should be conditionalized between upstream/downstream until this has happened.

Reply:
Conditionalized as requested.